includes/class-indieauth-authorize.php

```diff
@@ -108,12 +108,18 @@ public function http_header() {
 	public static function html_header() {
 		$auth  = static::get_authorization_endpoint();
 		$token = static::get_token_endpoint();
+		$kses  = array(
+			'link' => array(
+				'href' => array(),
+				'rel'  => array(),
+			),
+		);
 		if ( empty( $auth ) || empty( $token ) ) {
 			return;
 		}
 		if ( is_author() || is_front_page() ) {
-			printf( '<link rel="authorization_endpoint" href="%s" />' . PHP_EOL, $auth ); // phpcs:ignore
-			printf( '<link rel="token_endpoint" href="%s" />' . PHP_EOL, $token ); //phpcs:ignore
+			wp_kses( sprintf( '<link rel="authorization_endpoint" href="%s" />' . PHP_EOL, $auth ), $kses );
```

Comment:
This won't output anything, so should be this:
```suggestion
			echo wp_kses( sprintf( '<link rel="authorization_endpoint" href="%s" />' . PHP_EOL, $auth ), $kses );
```